docs/snippets/cli.rst

```diff
@@ -42,6 +42,14 @@ Usage
                             regular expression; default is --ignore-decorators=''
                             which does not ignore any decorated functions.
 
+.. note::
```

Comment:
A shorter version of this should also be added the to flag description (add it in the code, then call `pydocstyle --help` and paste the result above). It should be succinct, e.g., replace "with a list of comma separated error codes" with "with a list of comma separated error codes or prefixes".